proxy/src/config.rs

```diff
@@ -158,7 +158,7 @@ const DEFAULT_REPORT_TIMEOUT_SECS: u64 = 10; // TODO: is this a reasonable defau
 const DEFAULT_PRIVATE_LISTENER: &str = "tcp://127.0.0.1:4140";
 const DEFAULT_PUBLIC_LISTENER: &str = "tcp://0.0.0.0:4143";
 const DEFAULT_CONTROL_LISTENER: &str = "tcp://0.0.0.0:4190";
-const DEFAULT_PRIVATE_CONNECT_TIMEOUT_MS: u64 = 20;
+const DEFAULT_PRIVATE_CONNECT_TIMEOUT_MS: u64 = 200;
```

Comment:
Please pull this change out as a separate PR and let's land that PR right away to see if it might help with #478.

I doubt that 200ms is a long enough timeout. Why not 10 seconds like the other timeouts?

Reply:
Okay, I'd be happy to open a separate PR for this change.

200 ms was chosen fairly arbitrarily. I was assuming there was a reason for the original value being low and didn't want to increase the timeout too far beyond what was necessary. 50 and 100 ms were too low to fix the tests, at least on my box.

Reply:
I think 10 seconds is a much better timeout than what we have now. Imagine that the service is implemented in a GC'd language. Note here https://stackoverflow.com/questions/574021/reducing-jvm-pause-time-1-second-using-useconcmarksweepgc that somebody is asking how to reduce a GC pause of over 1 second. There's no way 200 ms will be sufficient.

Anyway, let's discuss it further in its own PR.

Reply:
10s seems completely reasonable to me.